File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1943.rb
n = gets.to_i
 
res = "Top "
if(n <= 1) then res += "1"
elsif(n <= 3) then res += "3"
elsif(n <= 5) then res += "5"
elsif(n <= 10) then res += "10"
elsif(n <= 25) then res += "25"
elsif(n <= 50) then res += "50"
else res += "100"
end

puts res

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1943.lua
n = io.read("*number")
 
res = "Top "
if(n <= 1) then res = res .. "1"
elseif(n <= 3) then res = res .. "3"
elseif(n <= 5) then res = res .. "5"
elseif(n <= 10) then res = res .. "10"
elseif(n <= 25) then res = res .. "25"
elseif(n <= 50) then res = res .. "50"
else res = res .. "100"
end

print(res)